src/store/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface AppState {
```

Comment:
I'm not really fluent in typescript & not sure that this is the best way to add types for store

Reply:
I'd add a new directory under src called `types` and name this file `types.d.ts`